chrome/content/zotero/xpcom/utilities_internal.js

```diff
@@ -519,6 +519,82 @@ Zotero.Utilities.Internal = {
 		
 		return deferred.promise;
 	},
+
+
+	/**
+	 * Takes in a document in a hidden web browser, creates a JS Sandbox and
+	 * executes the SingleFile extension to save the page as one single file
+	 * without JavaScript.
+	 *
+	 * @param {Object} document Document object inside hidden browser
+	 * @param {String|nsIFile} destFile Path for file to write to using Zotero.File
```

Comment:
Since the signature is changing anyway, this should only take a string path. nsIFile will be going away eventually.

No need to mention Zotero.File